src/ansible_builder/containerfile.py

```diff
@@ -106,6 +108,7 @@ def prepare(self) -> None:
                 "",
                 "# Galaxy build stage",
                 "FROM base as galaxy",
+                "ENV PIP_BREAK_SYSTEM_PACKAGES=1",
```

Comment:
this seem superfluous at a glance, since it's always set on base (even for pre-v3)

Reply:
Agreed! Will remove it before merge.